code/game/machinery/ARES/ARES_procs.dm

```diff
@@ -40,6 +40,12 @@ GLOBAL_LIST_INIT(maintenance_categories, list(
 	var/list/waiting_ids = list()
 	var/list/active_ids = list()
 
+	///Sentry faction stuff
+	var/faction_label = FACTION_MARINE
+	var/list/faction_group = FACTION_LIST_ARES_MARINE
+	var/list/faction_options = list(FACTION_MARINE, FACTION_WY, "USCM-WY", FACTION_ARES)
```

Comment:
why is one a string and the other defines?

Reply:
Because the defines are single factions and the string gives IFF to a faction list instead.

Reply:
Make this an assoc list for title - list of faction groups, otherwise fine